modules/social_features/social_landing_page/social_landing_page.module

```diff
@@ -271,34 +271,38 @@ function social_landing_page_preprocess_field(&$variables) {
 /**
  * Fetches the first available hero section image from a landing page.
  *
- * @param \Drupal\node\Entity\Node $node
+ * @param \Drupal\node\NodeInterface $node
  *   The landing page.
  *
  * @return array
  *   Render array of the image with a link.
  *
  * @throws \Drupal\Core\Entity\EntityMalformedException
  */
-function _social_landing_page_get_hero_image(Node $node) {
+function _social_landing_page_get_hero_image(NodeInterface $node): array {
```

Comment:
@dmytriikaiun Could you explain why did you make changes in this function, and how this is related to the permissions? 🤔

Reply:
@ribel It's not related to the permission. Just fixed warnings when we save the Hero block with an empty message. It takes few minutes and I think "Why not?"